test/term_test.go

```diff
@@ -47,7 +48,7 @@ func TestGetWinsize(t *testing.T) {
 		t.Fatal("winSize is nil")
 	}
 
-	newSize := Winsize{Width: 200, Height: 200, x: winSize.x, y: winSize.y}
+	newSize := Winsize{Width: 200, Height: 200}
```

Comment:
Ah, I think this was the bit I was still looking at. It's somewhat unclear why `x` and `y` aren't exported; effectively `WinSize` in this module is identical to golang.org/x/sys/unix.Winsize`;

Type in this module:

```go
// Winsize represents the size of the terminal window.
type Winsize struct {
	Height uint16
	Width  uint16
	x      uint16
	y      uint16
}
```

And in golang.org/x/sys:

```go
type Winsize struct {
	Row    uint16
	Col    uint16
	Xpixel uint16
	Ypixel uint16
}
```

In this module, we're converting the types between, so perhaps instead we should make it an alias (and deprecate the local type). Let me look at that.

Reply:
Oh! I recall now; for Windows, it's a different type, so we can make it an alias (but only on !windows)

Reply:
Hm.. nevermind; field names are different 🤦